README.md

```diff
@@ -96,24 +96,28 @@ deployment goals without "throwing the model over the fence and back".
   export OCTOML_PROFILE_API_TOKEN=<access token>
   ```
 
-- Create and activate a python virtual environment. Make sure that you are using python3.8.
+- Create and activate a python virtual environment. `python==3.8` is recommended and tested on both `ubuntu` and `macOS`. `python==3.10.9` is tested on `macOS` with M1 chip.
+
+  [venv]
 
-  [Linux]
-  using virtualenv:
   ```
   python3 -m venv env
-  . env/bin/activate
+  source env/bin/activate
   ```
+  or
 
-  [MacOS]
-  using conda:
-  ```
-  # Instructions on installing conda on MacOS: https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html
-  # Choose the Miniconda Installer for you Mac architecture.
+  [conda]
 
+  ```
   conda create -n octoml python=3.8
   conda activate octoml
   ```
+  if you are on Mac M1 and seeing `symbol not found in flat namespace '_CFRelease'`, it is likely that you created a `venv` with python installed by `conda`. Please make sure to deactivate any `conda` environment(s) by:
+  ```
+  conda config --set auto_activate_base false
```

Comment:
this line will change the default conda behavior which may not be desirable

Reply:
I also think if the user already has conda installed, it's probably lower friction to go through conda setup than switching back to venv.